lib/wickedbase.pm

```diff
@@ -984,6 +989,14 @@ sub post_run {
     $self->check_logs() unless $self->{skip_check_logs_on_post_run};
     $self->check_coredump();
     $self->upload_wicked_logs('post');
+
+    my $time_since_barrier_wait = time - ($self->{last_barrier_wait_call} // 0);
```

Comment:
this code is relevant ONLY for parent job , right ? Any chance that with current implementation child will also execute it ?

Reply:
we can simply guard it with `IS_WICKED_REF == 1`